src/components/TopBar/TopBar.vue

```diff
@@ -416,40 +410,13 @@ export default {
 		margin-right: var(--default-clickable-area);
 	}
 
-	&.in-call {
+	&[data-theme-dark="true"] {
 		right: 0;
 		border: none;
 		position: absolute;
 		top: 0;
-		left:0;
+		left: 0;
 		background-color: transparent;
-		display: flex;
-		flex-wrap: wrap;
-		& * {
-			color: #fff;
-		}
-
-		:deep(button.dark-hover:hover),
-		.dark-hover :deep(button:hover),
-		.dark-hover :deep(.action-item--open button),
-		:deep(.action-item--open.dark-hover button) {
-			background-color: rgba(0, 0, 0, 0.2);
-		}
-	}
-
-	&__button {
-		margin: 0 2px;
```

Comment:
I'm fine with it, just reflect in PR visual changes
![image](https://github.com/nextcloud/spreed/assets/93392545/eaab3f50-fcfa-445d-9a9a-565528dc97d5)

![image](https://github.com/nextcloud/spreed/assets/93392545/aba82b5e-9c8c-4a8e-bbbb-183cf24a8328)

Reply:
Although you know what, let's add a gap for `.top-bar`, at least 2-4px:

![image](https://github.com/nextcloud/spreed/assets/93392545/429a4b28-a89b-44cd-b96d-2ed543a5ac28)